projects/storefrontapp/src/environments/models/feature.model.ts

```diff
@@ -1,5 +1,5 @@
 import { ModuleWithProviders, Type } from '@angular/core';
 
 export interface FeatureEnvironment {
-  imports: Array<Type<any> | ModuleWithProviders<{}>>;
+  imports: (any[] | Type<any> | ModuleWithProviders<{}>)[];
```

Comment:
Why there is `any[]`?

Reply:
picked

Reply:
@Changsuwan Here you can just use the old, existing notation (i.e. revert our change).
This is not needed anymore

Reply:
Done